build/Dockerfile

```diff
@@ -14,36 +14,39 @@ RUN echo "4a7fa60f323ee1416a4b1425aefc37ea359e9d64df19c326a58953a97ad41ea5  go.t
 RUN tar -C /usr/local -xzf go.tar.gz
 ENV PATH=${PATH}:/usr/local/go/bin
 
+WORKDIR /build/windows-machine-config-operator/
+COPY .git .git
+
 # Build WMCB
-WORKDIR /build/windows-machine-config-bootstrapper/
+WORKDIR /build/windows-machine-config-operator/windows-machine-config-bootstrapper/
 COPY windows-machine-config-bootstrapper/ .
 RUN make build
 
 # Build hybrid-overlay
-WORKDIR /build/ovn-kubernetes/
+WORKDIR /build/windows-machine-config-operator/ovn-kubernetes/
 COPY ovn-kubernetes/ .
-WORKDIR /build/ovn-kubernetes/go-controller/
+WORKDIR /build/windows-machine-config-operator/ovn-kubernetes/go-controller/
 RUN make windows
 
 # Build promu utility tool, needed to build the windows_exporter.exe metrics binary
-WORKDIR /build/promu/
+WORKDIR /build/windows-machine-config-operator/promu/
 COPY promu/ .
-RUN make build
+RUN go build .
```

Comment:
@LorbusChris was this your suggestion? I thought we wanted to do `go install...`.